app/views/cart_js/reservation_form.js.erb

```diff
@@ -0,0 +1,11 @@
+resume_cart();
+
+<% if @errors.empty? or (can? :override, :reservation_errors) %>
+  $('#new_res_request').html('<%= escape_javascript(render :partial => '/reservations/new_reservation') %>');
```

Comment:
This is very minor but I'm not thrilled with this ID (`new_res_request`) - we use the term 'request' to mean something specific. What about something like `confirm-res-form` (also note the use of dashes instead of underscores for CSS selectors)? Just a suggestion...